packages/appsync-modelgen-plugin/src/preset.ts

```diff
@@ -217,6 +217,37 @@ const generateDartPreset = (
   return config;
 };
 
+const generateManyToManyModelStubs = (options: Types.PresetFnArgs<AppSyncModelCodeGenPresetConfig>) : TypeDefinitionNode[] => {
+  let models = new Array<TypeDefinitionNode>();
+  let manyToManySet = new Set<string>();
+  options.schema.definitions.forEach(def => {
+    if (def.kind === 'ObjectTypeDefinition') {
+      def?.fields?.forEach(field => {
```

Comment:
Can this be just `def.fields`? We use `def.kind` in the above conditional statement, so it must definitely have a value here.

Reply:
the compiler won't let it fly since it can still technically be undefined